src/envo/envo-edit.owl

```diff
@@ -31615,76 +31633,133 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001406> ObjectSomeValuesFrom(<
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000116> <http://purl.obolibrary.org/obo/ENVO_01001412> "Add more PATO qualities once they have been added to PATO."@en)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_01001412> <http://orcid.org/0000-0002-4366-3088>)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasAlternativeId> <http://purl.obolibrary.org/obo/ENVO_01001412> "Sc")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_01001412> "Haufenschichtwolke"@de)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_01001412> "Stratocumulus-Wolke"@de)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_01001412> "envoAtmo")
-AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_01001412> "Most of the regularly arranged small structural elements of statocumulus clouds have an apparent width of more than 5 degrees. Fibrous structural elements may emerge when virga is formed.")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_01001412> "envoCloudAtlas"@en)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_01001412> "When present in aggregates, individual stratocumulus clouds are organised in patches, sheets or layers. Most of the regularly arranged small elements in this cloud type often have an apparent width of more than five degrees. When well-illuminated by sunlight (excluding during sunrise and sunset) stratocumulus clouds appear grey or whitish, or both grey and whitish in colour. The cloud almost always has dark parts. Fibrous structural elements may emerge when virga is formed. Stratocumulus can have weak falls of rain, snow or snow pellets."@en)
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01001412> "stratocumulus cloud"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001412> <http://purl.obolibrary.org/obo/ENVO_01001428>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001412> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/ENVO_03520000>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001412> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002473> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/BFO_0000027> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002351> ObjectUnionOf(<http://purl.obolibrary.org/obo/ENVO_01000832> <http://purl.obolibrary.org/obo/ENVO_01000833> <http://purl.obolibrary.org/obo/ENVO_01001876> <http://purl.obolibrary.org/obo/ENVO_03520007>)))))
```

Comment:
I removed "water ice" from the `composed primarily of` axiom, as this cloud is mostly water. The other axiom directed towards the object aggregate still holds, though.